test/integration/kubeaggregator/aggregator_test.go

```diff
@@ -14,15 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package discoverysummarizer
+package kubeaggregator
```

Comment:
general question: how kube specific is all this? Not at all? Let's call it in a neutral fashion.